tsMuxer/abstractStreamReader.h

```diff
@@ -32,6 +32,7 @@ class AbstractStreamReader: public BaseAbstractStreamReader {
 	virtual int readPacket(AVPacket& avPacket) = 0;
 	virtual int flushPacket(AVPacket& avPacket) = 0;
 	virtual int getTSDescriptor(uint8_t* dstBuff) {return 0;}
+	virtual int getStreamHDR() const { return 0; }

```

Comment:
This causes `getStreamHDR` not to be abstract in all the classes deriving from `AbstractStreamReader`, notably `MPEGStreamReader` which redeclares the function as abstract. Is this intentional?

Reply:
@xavery you're correct: removed from AbstractStreamReader. First experience with C++, I'm still unfamiliar with inheritance...